bin/plotting/pycbc_faithsim_plots

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env python
+
+"""
+Program for plotting the results of the pycbc_faithsim and pycbc_collect_result script
+that compare two approximants and compute the match between them.
+"""
+
+import argparse
+import matplotlib
+
+matplotlib.use("Agg")
+import matplotlib.cm
+import pylab
+import numpy as np
+from ligo.lw import utils, table, lsctables
+from matplotlib.ticker import MultipleLocator, FormatStrFormatter
+from pycbc import pnutils
+from pycbc.conversions import (
+    mtotal_from_mass1_mass2,
+    q_from_mass1_mass2,
+    eta_from_mass1_mass2,
+    mchirp_from_mass1_mass2,
+    chi_eff,
+)
+
+
+def derived_quantities(data):
```

Comment:
This function is a bit inefficient, as it will always calculate every derived quantity even if you do not use it. I would use a dictionary of lambda functions here, something like this:
```python
derived_func_map = {
    "total_mass": lambda d: mtotal_from_mass1_mass2(d["mass1"], d["mass2"]),
    "mass_ratio": lambda d: q_from_mass1_mass2(d["mass1"], d["mass2"])
}
```
then you use it like so:
```python
elif args.parameter_x in derived_func_map:
    v1d = derived_func_map[args.parameter_x](data)
```